packages/react-components-pro/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@vaadin/react-components-pro",
+  "version": "24.4.0-alpha8",
```

Comment:
Let's also add this to ensure the new package is published with `--access public`:

```
  "publishConfig": {
    "access": "public"
  },
```

Reply:
Done